presto-main/src/test/java/io/prestosql/sql/planner/iterative/rule/TestDetermineJoinDistributionType.java

```diff
@@ -643,6 +655,185 @@ public void testReplicatesWhenNotRestricted()
                         values(ImmutableMap.of("B1", 0))));
     }
 
+    @Test
+    public void testReplicatesWhenSourceIsSmall()
+    {
+        VarcharType symbolType = createUnboundedVarcharType(); // variable width so that average row size is respected
+        int aRows = 10_000;
+        int bRows = 10;
+
+        // output size exceeds AUTOMATIC_RESTRICTED limit
```

Comment:
is there such a constant?

Reply:
no there isn't, but `AUTOMATIC_RESTRICTED` is used in other tests too. Let's make this cleanup a follow-up PR.